### PR TITLE
feat: configurable whitelist

### PR DESCRIPTION
**Pull request by 0xSulpiride**
## Description

- added `whitelistedEntities` configurable param to config.json
- added respective env vars `SKANDHA_[NETWORK]_WL_PAYMASTER`, `SKANDHA_[network]_WL_ACCOUNT`, `SKANDHA_[network]_WL_FACTORY` (a list of addresses separated with `,`)

Existing whitelist: https://github.com/etherspot/skandha/tree/1baf2cc10a167075d8333c3ee77fdaf70429dd58/packages/params/src/whitelisted-entities

## Types of changes
What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Further comments (optional)

-

### CODE REVIEW

File: packages/executor/src/services/UserOpValidation/validators/safe.ts [hunk 1]
```diff
@@ -255,15 +256,18 @@ export class SafeValidationService {
...
+          entityTitle as keyof IWhitelistedEntities
+        ];
       if (
         entityAddr &&
         whitelist != null &&
         // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-        whitelist[this.chainId] &&
-        whitelist[this.chainId]!.some(
-          (addr) => addr === ethers.utils.getAddress(entityAddr)
+        whitelist.some(
```
Comment (ch4r10t33r) [inline packages/executor/src/services/UserOpValidation/validators/safe.ts:267]: Should we also check if the whitelisted address is a smart contract?

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]
§

File: packages/executor/src/services/UserOpValidation/validators/safe.ts (hunk 1 above)
Comment (0xSulpiride) [reply packages/executor/src/services/UserOpValidation/validators/safe.ts:267]: no, the account might not be created yet during validation

**Comment by ch4r10t33r** [comment]
LGTM

**Review by ch4r10t33r — APPROVED** [review]

**Comment by ch4r10t33r** [comment]
@0xSulpiride Pls update README.md and provide necessary information @taylorferran on how to setup this configuration parameter.

**Pull request merged**